Prace_domowe/Praca_domowa4/grudzien_adrianna/pd4.ipynb

```diff
@@ -0,0 +1,1231 @@
+{
```

Comment:
<p>Fajnie jakby to wszystko było na CV policzone</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>